Makefile.doc

```diff
@@ -45,11 +45,17 @@ DOCCOMMON:=doc/common/version.tex doc/common/title.tex doc/common/macros.tex
 ######################################################################
 
 .PHONY: doc doc-html doc-pdf doc-ps
-.PHONY: stdlib full-stdlib
+.PHONY: stdlib full-stdlib docoq
 
 doc: sphinx stdlib
 
-sphinx: coq
+ifdef QUICK
+docoq:
+else
+docoq: coq
+endif
+
+sphinx: docoq
```

Comment:
@herbelin I am not Make expert but I do believe something such as:
```makefile
ifndef QUICK
SPHINX_DEPS := coq
endif

sphinx: $(SPHINX_DEPS)
```
to be more idiomatic makefile.

Reply:
Ok for this variant. Pushed.